### PR TITLE
public domain image examples for SSD outputs

### PR DESCRIPTION
**Pull request by aaronmarkham**
Just for evaluation and comments.

### CODE REVIEW

File: examples/ssd/example_outputs.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    ],
+    [
+      "dog",
+      591,
+      416,
+      976,
+      722
+    ],
+    [
+      "horse",
```
Comment (aaronmarkham) [inline examples/ssd/example_outputs.md]: does it matter that it picks up the wrong thing, or is that good for discussion?

File: examples/ssd/example_outputs.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+      662,
+      716,
+      1152
+    ]
+  ]
+}
+```
+
+### 3 Dogs on Beach
+![3 dogs on beach](https://farm9.staticflickr.com/8051/8081326814_991e7b15cc_o_d.jpg)
```
Comment (aaronmarkham) [inline examples/ssd/example_outputs.md]: I think it's an interesting, if not kind of funny, segmentation challenge, and it at least picks out three animals...

File: examples/ssd/example_outputs.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+  "prediction": [
+    [
+      "boat",
+      1177,
+      597,
+      1872,
+      2346
+    ],
+    [
+      "boat",
```
Comment (aaronmarkham) [inline examples/ssd/example_outputs.md]: I wish we could plot these, because there is a tiny boat out there...

Comment (aaronmarkham) [reply examples/ssd/example_outputs.md]: downres of the file fixed that...

File: examples/ssd/README.md [hunk 4]
```diff
@@ -13,14 +13,14 @@ The inference service would return the response in the format - '[(object_class,
...
- wget https://s3.amazonaws.com/model-server/models/resnet50_ssd/resnet50_ssd_model-symbol.json
- wget https://s3.amazonaws.com/model-server/models/resnet50_ssd/resnet50_ssd_model-0000.params
+ curl -O https://s3.amazonaws.com/model-server/models/resnet50_ssd/resnet50_ssd_model-symbol.json
+ curl -O https://s3.amazonaws.com/model-server/models/resnet50_ssd/resnet50_ssd_model-0000.params
 ```
 
-or
-
-Use these links to download the Symbol and Params files:
+Alternatively, use these links to download the Symbol and Params files via your browser:
```
Comment (jesterhazy) [inline examples/ssd/README.md]: nit: do we usually capitalize these names?

**Pull request merged**